licenses/LICENSE-bootstrap.txt

```diff
@@ -145,21 +145,21 @@
           aria-label="Search this repository"
           data-unscoped-placeholder="Search or jump to…"
           data-scoped-placeholder="Search or jump to…"
-          data-jump-to-suggestions-path="/_graphql/GetSuggestedNavigationDestinations#csrf-token=prfvTpHCd6KRAPr1SmcgvMGbXslWfxLGZTMSbXIQUE/vax5H6vA0IJ7ix288dM5mdzJkUIqoMK2J4GvPPBkMZg=="
```

Comment:
Oh wow. This entire file shouldn't have even been touched.

Also I don't know what I put here but it clearly isn't even a LICENSE.... I'll open an issue for this.

Reply:
Thanks for opening my eyes to this -- I've opened #79 to address this.